chromium_src/chrome/browser/ui/webui/settings/about_handler.cc

```diff
@@ -2,8 +2,4 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "brave/browser/version_info.h"
-
-#define GetVersionNumber GetBraveVersionNumberForDisplay
 #include "../../../../../../../chrome/browser/ui/webui/settings/about_handler.cc"
```

Comment:
if nothing is being used here- we can remove this file altogether 😄 This was being used to force the webui code to register a handler (called on the Polymer page) for `GetBraveVersionNumberForDisplay` (see https://github.com/brave/brave-core/blob/9242b589bb103119ec0fa1ec66af61f5da3c4294/browser/version_info.cc#L14-L17) in place of `GetVersionNumber`

NOTE: Although we can delete this `about_handler.cc` override, we can't delete `GetBraveVersionNumberForDisplay` because it's used in brave://version for formatting the version


I think this formatting (as you have it) doesn't quite match the spec. Given your screenshot, I think it should instead have:
```
[hyperlink]Brave 1.5.76 (Developer Build) (64-bit)[/hyperlink]
Chromium 80.1.5.76
```

You should be able to adjust the logic in `brave_settings_overrides.js` to get that working 😄

Reply:
> if nothing is being used here- we can remove this file altogether
Sorry yeah first PR into the cpp code, I didn't realise how the linking happened until afterwards.
I noticed the other usage of `GetBraveVersionNumberForDisplay` could probably be removed as a follow up PR if we wanted.